packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/fixtures/MongoDbEnvironmentTestExtensions.kt

```diff
@@ -147,6 +147,16 @@ internal class DirectMongoDbDriver(
         throw UnsupportedOperationException()
     }
 
+    override suspend fun <T : Any, S> runQuery(
+        query: Node<S>,
+        result: KClass<T>,
+        queryContext: QueryContext,
+        timeout: Duration,
+        limit: Int
+    ): QueryResult<T> {
+        throw UnsupportedOperationException()
+    }
+
     override suspend fun <T : Any> runCommand(
```

Comment:
I think it's better to implement this during INTELLIJ-207 because if not, we would generate queries that won't run.